--- a/wled00/const.h
+++ b/wled00/const.h
@@ -1644,1 +1644,1 @@
-#define WLED_MIN_VIRTUAL_BUSSES 6
+#define WLED_MIN_VIRTUAL_BUSSES 21
--- a/wled00/wled.h
+++ b/wled00/wled.h
@@ -2819,1 +2819,1 @@
-WLED_GLOBAL int ethernetType _INIT(WLED_ETH_NONE); // use none for ethernet board type if default not defined
+WLED_GLOBAL int ethernetType _INIT(WLED_ETH_WT32_ETH01); // custom build: default WT32-ETH01
